backend/src/main/java/com/eyetrade/backend/repository/UserRepository.java

```diff
@@ -8,15 +8,15 @@
 
 @Repository
 public interface UserRepository  extends JpaRepository<User,String> {
+
     User findByEmail(String email);
 
     User findByEmailAndPword(String email, String pword);
 
     User findUserByIdentityNo(String identityNo);
 
-    User findByIdentifier(String identifier);
```

Comment:
It will be used  in future probably, because we will develop over id from a point. So we can keep this method for future